docs/guides/use-git-https.md

```diff
@@ -0,0 +1,60 @@
+# Using Git over HTTPS
+
+Instead of making use of Flux' capabilities to generate an SSH private
+key, or [supplying your own](provide-own-ssh-key.md). It is possible to
+set the `GIT_AUTHUSER` and `GIT_AUTHKEY` environment variables and use
+these in your `--git-url` argument to provide your HTTPS basic auth
+credentials without having to expose them as a plain value in your
+workload.
+
+> **Note:** supplying the `GIT_AUTHKEY` in the `--git-url` will
+> disable the generation of a private key and prevent the setup
+> of the SSH keyring.
+
+> **Note:** the variables _must be escaped with `$()`_ for Flux
```

Comment:
It's not Flux that requires the parens, it's Kubernetes. So a better explanation here might be

> env variables used in Kubernetes manifests are escaped with `$()`